### PR TITLE
Bowling: Add new test

### PR DESCRIPTION
**Pull request by isgj**
Following exercism/rust#916 I'm adding this test.

### CODE REVIEW

File: exercises/bowling/canonical-data.json [hunk 1]
```diff
@@ -240,5 +240,12 @@
       "roll": 2
     },
     "expected": {"error": "Cannot roll after game is over"}
+  }, {
```
Comment (cmccandless) [inline exercises/bowling/canonical-data.json]:
```suggestion
  },
  {
```

**Comment by rpottsoh** [comment]
Yes this repo is still not accepting changes along the lines proposed by this PR.

However, there are two changes I would like to see made to this PR.

1.  Line 3: Version needs to be incremented per rules in [README](https://github.com/exercism/problem-specifications#test-data-versioning).
1.  I would like to see the new case moved up so that it is grouped with the other `Score` cases similar to this one.

Someday the freeze will be lifted and another good change will be ready to go.  👍

**Comment by ErikSchierboom** [comment]
@isgj Are you still interested in working on this PR?

**Comment by ErikSchierboom** [comment]
CC @exercism/reviewers maybe someone can review?

**Review by kotp — APPROVED** [review]
It looks right to me, and @petertseng reviewed it as it applies to Rust as well.

**Review by ErikSchierboom — APPROVED** [review]
My only comment would be to perhaps move this test case up a bit, as we usually try to order the test cases by order of increasing difficulty.

**Review by ErikSchierboom — CHANGES_REQUESTED** [review]
It looks like the JSON is invalid: https://github.com/exercism/problem-specifications/runs/4963687045?check_suite_focus=true

**Comment by ErikSchierboom** [comment]
@isgj Thanks for the quick fix! There are two minor issues remaining:

1. It think it would make sense to move this new test case to https://github.com/exercism/problem-specifications/blob/ecf797f3086f9b70dec11d211e419ba02fcb8ad5/exercises/bowling/canonical-data.json#L107, along with the other valid bonus roll test cases. Would you be okay moving the test case?
2. The CI is failing because since this PR was opened, we introduced a `uuid` property to all test cases. Maybe you could rebase this PR on the latest `main` and add the `uuid` property to the new test case? You can generate one using https://www.uuidgenerator.net/version4. Sorry about that.

**Review by ErikSchierboom — APPROVED** [review]
Excellent. LGTM!

**Review by kotp — APPROVED** [review]

**Review by petertseng — APPROVED** [review]

**Comment by kotp** [comment]
@isgj Thank you very much for the work done with this!

**Pull request merged**